src/OrchardCore.Themes/TheAdmin/wwwroot/Scripts/TheAdmin.js

```diff
@@ -2629,7 +2629,7 @@ function _typeof(obj) { "@babel/helpers - typeof"; if (typeof Symbol === "functi
 function _typeof(obj) { "@babel/helpers - typeof"; if (typeof Symbol === "function" && typeof Symbol.iterator === "symbol") { _typeof = function _typeof(obj) { return typeof obj; }; } else { _typeof = function _typeof(obj) { return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj; }; } return _typeof(obj); }
 
 /*!
-  * Bootstrap v4.5.3 (https://getbootstrap.com/)
+  * Bootstrap v4.5.2 (https://getbootstrap.com/)
```

Comment:
This PR regressed the bootstrap version.

Reply:
Nah, all the time happen to me, when I run a gulp, after a pull

Reply:
You need to npm install after a pull.

Reply:
I'm doing npm run build ;) seems that's the issue, frankly I'm not a npm familiar guy